src/content/en/fundamentals/security/prevent-mixed-content/what-is-mixed-content.md

```diff
@@ -14,18 +14,19 @@ description: Mixed occurs when initial HTML is loaded over a secure HTTPS connec
 connection, but other resources (such as images, videos, stylesheets, scripts)
 are loaded over an insecure HTTP connection. This is called mixed content
 because both HTTP and HTTPS content are being loaded to display the same page,
-and the initial request was secure over HTTPS. Modern browsers display warnings
-about this type of content to indicate to the user that this page contains
-insecure resources.
+and the initial request was secure over HTTPS. 
+
+Modern browsers may block this content, or will display warnings about this type of content to indicate to the user that this page contains insecure resources. Browsers that block mixed content may first attempt to "upgrade" the connection to this content from HTTP to HTTPS.
```

Comment:
The last sentence refers to "Browsers" but really is just in reference to Chrome's plan. Not sure how these articles like to refer to single-browser initiatives.

Reply:
I'm also not sure what the right precedent is, but fwiw this sounds good to me. We could maybe link to https://www.chromestatus.com/feature/5557268741357568 and https://www.chromestatus.com/feature/4926989725073408 so people can get more up-to-date information about the specifics of what browsers upgrade what content.

Reply:
This is under /web, so I'm not too concerned about it being primarily Chrome focused.